public/codenotes.php

```diff
@@ -148,21 +162,30 @@ function saveCodeNote(rowIndex) {
         // Before bailing from edit mode, make sure all new values for
         // the note (the text content and author avatar) are properly
         // displayed in the UI.
-        const noteValueWithLineBreaks = noteEditEl.value.replace(/\n/g, '<br />');
-        noteDisplayEl.innerHTML = noteValueWithLineBreaks;
+        if (isDeleting) {
```

Comment:
Can this be moved up somehow? Guarding every previous conditional expression with !isDeleting etc. is a bit awkward.